src/euphorie/client/browser/training.py

```diff
@@ -238,6 +239,23 @@ def session(self):
         """Return the session for this context/request"""
         return self.context.session
 
+    def get_initial_answers(self):
+        """Pick a subset of questions, shuffle them, and initialize the answers with
+        None.
+        """
+        survey = self.webhelpers._survey
+        all_questions = api.content.find(
+            **{
+                "portal_type": "euphorie.training_question",
+                "path": "/".join(survey.getPhysicalPath()),
+            }
```

Comment:
```suggestion
            obj=survey,
            portal_type="euphorie.training_question",
```

Reply:
I wasn't aware of the `obj=` shorthand. That's handy, cheers.